### PR TITLE
Handle outliers

### LINKED ISSUE #8: Write function to handle outliers
> We will check all data for biologically implausible values, and for significant outliers by examining if they fall ±4 standard deviations from the mean. These values will be excluded from the primary analyses, with the full, unrestricted dataset included as a sensitivity analysis. 

e.g.

handle_outliers <- function(x) { ... }

### PR DESCRIPTION
**Pull request by conig**
Simple function to replace outliers with NA, and a helper function to apply it to a vector of column names.
When data cleaning done, should be simple to apply this enmass to variables as per issue #8

### CODE REVIEW

File: R/cleaning/remove_outliers.R [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#'
+#' This function removes outliers from a numeric vector by replacing values 
+#' that are more than 4 standard deviations from the mean with NA.
+#'
+#' @param x A numeric vector
+#'
+#' @return A numeric vector with outliers replaced by NA
+#'
+#' @examples
+#' x <- c(1, 2, 3, 100)  # 100 is an outlier
```
Comment (tarensanders) [inline R/cleaning/remove_outliers.R]: Not that it matters, but this example doesn't work.

File: R/cleaning/remove_outliers.R [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#' remove_outliers(x)
+#' 
+remove_outliers <- function(x) {
+ x[scale(x) > 4] <- NA
```
Comment (tarensanders) [inline R/cleaning/remove_outliers.R]: Should be 

`x[scale(x) > 4 & scale(x) < -4] <- NA`

right?

Comment (conig) [reply R/cleaning/remove_outliers.R]: ah yeah whoops, abs() also would have done it.

File: R/cleaning/remove_outliers.R (hunk 1 above)
Comment (conig) [reply R/cleaning/remove_outliers.R]: Fixed.

**Comment by conig** [comment]
Errors should now be fixed. Thanks for catching, and sorry, that was dodge.

**Review by tarensanders — APPROVED** [review]
Thanks for changing that

**Pull request merged**